app/models/project.rb

```diff
@@ -35,6 +35,18 @@ def self.sponsored_projects(sponsor)
     Project.where("metadata_json->>'data_sponsor' = ?", sponsor)
   end
 
+  def self.managed_projects(manager)
+    Project.where("metadata_json->>'data_manager' = ?", manager)
+  end
+
+  def self.data_user_projects(user)
+    # See https://scalegrid.io/blog/using-jsonb-in-postgresql-how-to-effectively-store-index-json-data-in-postgresql/
+    # for information on the @> operator
+    query_ro = '{"data_user_read_only":["' + user + '"]}'
+    query_rw = '{"data_user_read_write":["' + user + '"]}'
+    Project.where("(metadata_json @> ? :: jsonb) OR (metadata_json @> ? :: jsonb)", query_ro, query_rw)
```

Comment:
We are no sure this is the best SQL to query both fields, but it does the job.

Reply:
First get it working, then later get it working fast if performance is a problem. :) 
LGTM.

Reply:
Also, hooray for the inline comment pointing to documentation.